packages/synapse-react-client/src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -296,45 +299,47 @@ export const DownloadConfirmation: React.FunctionComponent<
   return (
     <>
       <Alert
```

Comment:
Needs a little padding IMO
```suggestion
      <Alert
        sx={{
          pr: 4,
          py: 1,
        }}
```

Reply:
Added in next commit